metadata/service/server/init.yml

```diff
@@ -0,0 +1,24 @@
+applications:
+- rsyslog
+classes:
+- service.rsyslog.support
+parameters:
+  _param:
+    rsyslog_port: 10514
+  rsyslog:
+    server:
+      enabled: true
+      module:
+        imtcp: {}
+      template:
+        RemoteStorage:
+          parameter:
+            type: string
+            string: /var/log/%HOSTNAME%/%programname%.log
+      ruleset:
+        RemoteLogs:
+          description: action(type="omfile" dynaFile="RemoteStorage")
```

Comment:
Now I noticed these rules has standard format, we could consider using following instead:
```
ruleset:
  RemoteLogs:
    action:
      type: omfile
      dynaFile: remoteStorage
```

that would result in much cleaner and readable metadata